install/streams.csv

```diff
@@ -2131,10 +2131,9 @@ TSZ,10,0.1,false,false,false,2004-01-26T00:00:00Z,2013-12-31T00:00:00Z
 TSZ,10,1,false,false,false,2004-01-26T00:00:00Z,9999-01-01T00:00:00Z
 TSZ,10,100,false,false,false,2004-01-26T00:00:00Z,9999-01-01T00:00:00Z
 TSZ,20,1,false,false,false,2004-01-26T00:00:00Z,2013-07-01T23:00:00Z
-TSZ,20,50,false,false,false,2013-07-02T21:54:34Z,2019-11-21T22:58:00Z
-TSZ,20,200,false,false,true,2004-01-26T00:00:00Z,2019-11-21T22:58:00Z
-TSZ,21,50,false,false,false,2019-11-21T22:59:00Z,2020-06-16T00:00:00Z
-TSZ,21,200,false,false,true,2019-11-21T22:59:00Z,9999-01-01T00:00:00Z
+TSZ,20,200,false,false,true,2004-01-26T00:00:00Z,2013-07-01T23:00:00Z
+TSZ,21,50,false,false,false,2013-07-02T21:54:34Z,2020-06-16T00:00:00Z
+TSZ,21,200,false,false,true,2013-07-02T21:54:34Z,9999-01-01T00:00:00Z
```

Comment:
- close LC 21 at 200hz triggered on 2020-02-08/23:59 and open LC 21 200hz continuous on 2020-02-09/00:00?

Reply:
Done Miss :)